launchpad_backend/launchpad_server/routes/index.py

```diff
@@ -6,7 +6,8 @@
 from flask_pymongo import PyMongo
 import pymongo
 import sys
-sys.path.append('.\\launchpad_server\\routes')
+import re
+sys.path.append('./launchpad_server/routes')
```

Comment:
can someone check if this works on a windows environment? The \\ wasn't working on my laptop.

Reply:
LGTM